README.md

```diff
@@ -376,6 +378,50 @@ curl -XPUT https://xxxxxxx.execute-api.us-east-1.amazonaws.com/dev/dynamodb/<has
  -H 'Content-Type:application/json'
 ```
 
+
+### EventBridge
+
+Sample syntax for EventBridge proxy in `serverless.yml`.
+
+```yaml
+custom:
+  apiGatewayServiceProxies:
+    - eventbridge: # source and detailType is set to apigateway requestid by default
+        path: /eventbridge
+        method: post
+        eventBusName: { Ref: 'YourBusName' }
+        cors: true
```

Comment:
`source` may be better to be required property. EventBridge can route events based on `source`, therefore I think there is no use case that set auto-generate string. What do you think?

Reply:
Yes, that actually makes sense. Will make changes to make source required.

Reply:
Thanks 👍